### DISCUSSION TITLE
在配置了msys2的环境下疑似在test snippet的时候错误包含了msys2的包含文件夹

### DISCUSSION THREAD

**Discussion by RimuruChan** [xmake-io/xmake#4722] [Q&A]
![image](https://github.com/xmake-io/xmake/assets/42002296/1dcab4d9-5105-49b4-97ad-642ac1099c76)
最终导致编译失败

**Comment by waruqi** [comment]
检查下，是否引入的某个包 找到了系统库，被 pkgconfig 给引入了，或者配置 system = false 走仓库版本
